GetJobber/atlantis
# docs(docx): Adding Hooks to new docs site [JOB-111817]

![Screenshot 2024-12-17 at 12 45 56 PM](https://github.com/user-attachments/assets/0340a2a1-c295-4888-a342-a71ad9accdfe)
![Screenshot 2024-12-17 at 12 52 55 PM](https://github.com/user-attachments/assets/c4e5386e-f5c4-4282-8738-ecc714681611)


### Added

- Added Hooks pages to the new docs site
- Added Hooks pages to sidenav
- Added Hooks collection page

### Changed

- In order to get examples to display properly, all usages of `<Story>` in the hook storybook stories files needed to be removed. The Story's inner contents have been moved to a separate component file and then imported to get around this.

## Testing

On the new docs site:
- ensure hooks are in left side nav and links work
- ensure you can get to the hooks collection page and links work
- check some hooks pages and make sure things display as expected

On the old docs site:
- make sure hooks pages load as expected

Changes can be
[tested via Pre-release](https://github.com/GetJobber/atlantis/actions/workflows/trigger-qa-build.yml)

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
